test/conformance/api/v1/service_test.go

```diff
@@ -581,3 +582,56 @@ func TestAnnotationPropagation(t *testing.T) {
 		t.Error("Route still has `juicy` annotation")
 	}
 }
+
+// TestServiceCreateWithMultipleContainers tests both Creation paths for a service.
+// The test performs a series of Validate steps to ensure that the service transitions as expected during each step.
+func TestServiceCreateWithMultipleContainers(t *testing.T) {
+	if !test.ServingFlags.EnableAlphaFeatures {
+		return
```

Comment:
This should probably be a `t.Skip()` instead?

Reply:
modified 

Thank you